src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -213,4 +222,121 @@ bool SetMaxOpenFilesSoftLimit(uint64_t soft_limit) {
   return true;
 }
 
+// Check for the existence of a file mapping containing the absolute address of the function. We
```

Comment:
So we would only report functions that we can not instrument at all. Wouldn't it be nicer to report the once where we fail for some mappings? Otherwise, I might miss calls that I would expect.

Reply:
Hmm I didn't think about it (having a module mapped multiple times is a case I still don't fully comprehend). So you suggest to change this function to return true only if a file mapping containing the absolute address is found for all the `ModuleInfo`s?

Reply:
I don't think it would be restricted to only one file mapping.

I would change it to only return true for a given function, if all modules that contain this function are mapped as a file mapping.

Reply:
> I don't think it would be restricted to only one file mapping.

I didn't get this part, because from the next one I feel like we are saying the same thing. Could you clarify?

Reply:
I changed this function (and renamed it) to return true (if and) only if a file mapping containing the absolute address of the function exists for all occurrences of the module. Please have a look and let me know if this is what you were thinking of.